hazelcast/src/test/java/com/hazelcast/internal/partition/TestPartitionUtils.java

```diff
@@ -208,14 +204,9 @@ public void run() {
             latch.countDown();
         }
 
-        private void await() throws InterruptedException {
-            Assert.assertTrue("GetReplicaVersionsRunnable is not executed!", latch.await(1, TimeUnit.MINUTES));
```

Comment:
This `latch.await()` was the cause for a ton of `throws InterruptedException` in the test code. I replaced it with `assertOpenEventually()`, which makes it much easier to use the utility method `getDefaultReplicaVersions()`.